Janus-build.bat

```diff
@@ -8,8 +8,13 @@ SET Feature_AlwaysDefaultVendor=false
 SET _VENDOR_=JAN
 ::version for Win10 1.0.N.100
 SET JAN_RELEASE_N=1
-call tools\build.bat vioserial\vioser.sln Win10 x64
-if errorlevel 1 goto :eof
+set PROJS=vioserial\vioser.sln ivshmem\ivshmem.sln
```

Comment:
Please check that ivshmem have proper build config for `SET _VENDOR_=JAN` and set valid copyright info

Reply:
Checked, everything is OK